geomstats/geometry/grassmannian.py

```diff
@@ -78,3 +84,34 @@ def exp(self, vector, point):
         expm = gs.linalg.expm
         mul = MatricesSpace.mul
         return mul(mul(expm(vector), point), expm(-vector))
+
+    def log(self, point, base_point):
+        """
+        Riemannian logarithm of a point from a base point.
+
+        Returns a skew-symmetric matrix in the image of [so(n), point].
+
+        Parameters:
+        ----------
+        point : array-like, shape=[n_samples, n, n]
+        base_point : array-like, shape=[n_samples, n, n]
+
+        Returns:
+        -------
+        log : array-like, shape=[n_samples, n, n]
+        """
+        svd = gs.linalg.svd
+        mul = gs.matmul
+        tr = MatricesSpace.transpose
+        logm = gs.linalg.logm
+
+        def closest(rot):
```

Comment:
Closest to rot, in the grassmanian, for its own metric? maybe add a docstring so that it's easier for us to understand quickly what is being computed.

Reply:
Hmm... as the name suggests `closest(rot)` is supposed to yield the closest representant of `rot` in SO(n) up to SO(k) x SO(n-k) 
But I realized this was actually a hack and although it does yield an interpolating curve in Gr(k, n) this curve is not ensured to be geodesic.